package.json

```diff
@@ -49,7 +49,7 @@
   },
   "main": "./src/fake-timers-src.js",
   "dependencies": {
-    "@sinonjs/commons": "^1.7.0"
+    "@sinonjs/commons": "2.0.0"
```

Comment:
why not semver range?

Reply:
That's a mistake on my part. I'll fix that in `main`

Reply:
https://github.com/sinonjs/fake-timers/pull/447